includes/wikia/DefaultSettings.php

```diff
@@ -1797,3 +1798,8 @@
  * flag for using (possibly external) service for managing a user's preferences
  */
 $wgPreferencesUseService = false;
+
+/**
+ * Parser Tag
+ */
+require_once "$IP/extensions/wikia/SoundCloudTag/SoundCloudTag.setup.php" ;
```

Comment:
I would set a $wgEnableXXXExt = true in here and then include the setup file in CommonExtensions as usual. I'm not the expert though, since @owend and @macbre ;-)